airbyte-webapp/src/views/layout/SideBar/components/MenuContent.tsx

```diff
@@ -0,0 +1,13 @@
+import { PropsWithChildren } from "react";
+
+import styles from "./MenuContent.module.scss";
+
+export const MenuContent: React.FC<PropsWithChildren<{ children: React.ReactNode[] }>> = ({ children }) => {
+  return (
+    <ul className={styles.menuContent}>
+      {children.map((child, index) => {
```

Comment:
Not a huge fan of this approach, since overriding the `children` property like this means this component can't be used with a single child, which seems weird. We could also remove the `PropsWithChildren` helper, since we are overriding the `children` type it provides anyway.

Reply:
Fair enough!  I think it's better to keep the helper as it sounds important for an eventual React18 upgrade.  Switched things up so it supports one or more children here.

Reply:
I think for the React 18 upgrade it's only important that `children` has _some_ sort of type, so it would be fine to remove here. If you get rid of it completely (just use `React.FC`) you'll notice that `children` still has the correct type:

<img width="591" alt="Screenshot 2023-02-13 at 9 43 15 PM" src="https://user-images.githubusercontent.com/7550957/218570378-8ca50a09-c562-47db-8fac-d0ef3728fa6b.png">

In React 18 this ^ will not work, since the implicit typing of `children` has been removed. But since you are defining it manually anyway, no need to use the `PropsWithChildren` helper type here (under the hood, it's just `type PropsWithChildren<P> = P & { children?: ReactNode | undefined };`) More info here if you're interested! https://solverfox.dev/writing/no-implicit-children/

Reply:
Got it!  I instead kept `PropsWithChildren` and added an array check before the map... which I think should solve it.